### PR TITLE
6-spec-getinvolved

### PR DESCRIPTION
**Pull request by shane-jeon**
## Ticket Description
<-- Copy and paste or write the tickets description here-->
- GetInvolved, mobile and browser responsive. Included main card and subcards for browser, responsiveness seems functional (hidden when size goes to md and below) Mobile carousel functional (code hidden once size goes to lg), styling reflective of figma design, however assistance is needed in making all mobile carousel slides uniform in height (possibly width). Mobile code starts from line 225.

## Description of Changes
<-- Brief description of the changes you made, files you touched, etc -->
- Made file "GetInvolved.tsx" in components. Added placeholder for component in home.tsx. Branch is up-to-date with main.

## Before and After for UI Updates
<-- You should have 2 images minimum, for desktop, 4 for mobile and desktop combined -->

Before:
N/A

After:
browser view
Screen Shot 2023-04-25 at 11 15 45 AM

mobile view (set to iPhone 12 pro)
Screen Shot 2023-04-25 at 11 22 29 AM
Screen Shot 2023-04-25 at 11 22 35 AM
Screen Shot 2023-04-25 at 11 22 41 AM
Screen Shot 2023-04-25 at 11 22 48 AM

## For PR Reviewer
- [ ] Does this file change the yarn.lock, package.json or package-lock.json file? If so, why?
- [ ] If this pr contains mobile and desktop changes, did you test on IphoneXr and desktop views?
- [ ] Does this file match the related tickets linked figma file, or does it pass the visual smell test?
- [ ] If this file contains javascript, does the javascript pass the smell test?
- [ ] If you don't feel super confident in your review, did you assign someone more senior to double check?

### CODE REVIEW

File: pages/_document.tsx
Comment (najwanan) [inline pages/_document.tsx:1]: It doesn't look like there are changes to this file, but we'll have to format out the extra spaces. There's a shortcut to fixing all linting errors. You won't have to do it manually. But this will get caught once we incorporate super linter through github actions. 

TLDR: We'll have to fix the linter errors. There's a shortcut npm command to do this through the terminal. I'll drop instructions in the main channel

File: components/GetInvolved.tsx
Comment (najwanan) [inline components/GetInvolved.tsx:1]: Starting on line 42, there's several linting issues that we'll need to fix. Not a big deal, I resolved it locally by hitting command and "." on my Mac. Wondering if you can do the same.

**Review by najwanan — COMMENTED** [review]
@shane-jeon This is really good work. 

I see what you mean about the cards not being of equal length (in mobile view). I think this may be something we reserve for cleanup. 

Some of the more immediate problems are:
1.) There's a back-to-the-top button that's currently very large in scale and stuck at the top of the screen. Not sure if this was part of this PR. 
2.) Merge conflicts need to be resolved. 
3.) Linting errors need to be fixed

If we can get these fixed I think this PR is ready to merge.

**Comment by najwanan** [comment]
@shane-jeon thanks for pushing up the latest changes! I'm working on getting the remaining PRs merged in. You'll be able to see your work shortly!
